src/Utils/FileSystem.php

```diff
@@ -135,7 +135,7 @@ public static function write(string $file, string $content, ?int $mode = 0666):
 	{
 		static::createDir(dirname($file));
 		if (@file_put_contents($file, $content) === false) { // @ is escalated to exception
-			throw new Nette\IOException("Unable to write file '$file'. " . Helpers::getLastError());
+			throw new Nette\IOException("Unable to write file '$file' with mode " . decoct($mode) . '. ' . Helpers::getLastError());
```

Comment:
$mode is not realated to file_put_contents()

Reply:
I will fix it, sorry.

Reply:
Fixed.